submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,161 @@
+const NUM_OF_FRIENDS = 20;
+const FRIENDS = document.querySelector(".friends");
+let allFriends;
+let friendsForFilter;
+let isFiltered = false;
+let justMale;
+let justFemale;
+
+function fetchFriends() {
+  const api_url = `https://randomuser.me/api/?results=${NUM_OF_FRIENDS}`;
+  fetch(api_url)
+    .then(handleErrors)
+    .then((response) => response.json())
+    .then((data) => (allFriends = data.results))
+    .then(() => addFriends(allFriends))
+    .catch((error) => showErrorMessage());
+}
+
+function handleErrors(response) {
+  if (!response.ok) {
+    throw Error(response.statusText);
+  }
+  return response;
+}
+
+function showErrorMessage() {
+  const wholePage = document.querySelector("body");
+  wholePage.setAttribute("class", "refresh");
+  wholePage.innerHTML = `<div>Ops...Something went wrong.</div>
+        <div>Refresh the page to start finding friends</div>`;
+}
+
+function addFriends(friendsToBeAdded) {
+  let fragment = document.createDocumentFragment();
+  friendsToBeAdded.forEach((friend) => {
+    let friendCard = document.createElement("div");
+    friendCard.setAttribute("class", "friend_card");
+    friendCard.innerHTML = `<div class="card_wrapper ${
+      friend.gender
+    }"><div class="name ">${`${friend.name.first} ${friend.name.last}`}</div>
+                                <div class="photo"><img src="${
+                                  friend.picture.large
+                                }"></div>
+                                <div class="info_block">
+                                <div class="age">Age ${friend.dob.age}</div>
+                                <div class="place">${friend.location.city}</div>
+                                <div class="email"><a href="mailto:${
+                                  friend.mail
+                                }" class="email_link">${friend.email}</a></div>
+                                <div class="gender">${friend.gender}</div>
+                                </div></div>
+                                `;
```

Comment:
Khm :) https://github.com/kottans/frontend-2021-homeworks/pull/434#discussion_r565419493

Looks like you forgot to add indentations.

It should be something like: 

```html
<div>
  <div></div>
</div>
```

Reply:
My bad. I've used Prettier incorrectly.
Fixed this by extended number of symbols in one line.